tests/test_edgeql_expressions.py

```diff
@@ -3057,7 +3057,7 @@ class TestExpressions(tb.QueryTestCase):
 
             bb';
             ''',
-            ['bbaa bb'],
+            ['bbaa \n\n            bb'],
```

Comment:
Actually, I strongly disagree with this change. The whole point of our `\` is that is consumes all white-space until next non-white-space, thus allowing to format a single logical string as you like. Consider this:
```
SELECT 'Always remember that: "\

        Long strings are very \
        useful because they let \
        the client make rendering \
        decisions.\

        " is all in one line (or, perhaps, \
        one paragraph)';
```

The point is that we wanted to find a way to make our multi-line strings somewhat easier to format as single logical line and only folding a single line after `\` is not that helpful. In practice I would never use that feature if I had several newlines, but as my example above shows I would be quite happy with the option of visually separating text blobs with newlines while keeping them all a contiguous string.

@1st1, @elprans what do you think about handling whitespace after `\`?

Reply:
I also disagree with the change. Let's handle `\` the way rust handles them:

![Screen Shot 2020-03-05 at 12 17 27 PM](https://user-images.githubusercontent.com/239003/76022341-c984e500-5edb-11ea-8f08-2da9df1559a1.png)

Reply:
That means that the behavior we implemented in a2 and current master branch is the correct one.

Reply:
Okay. Fixed that